packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-line.tsx

```diff
@@ -88,7 +89,7 @@ export const MovableLine = (props: Props) => {
                         stroke,
                         strokeWidth: "var(--movable-line-stroke-weight)",
                     }}
-                    dragging={dragging}
+                    className={dragging ? "movable-dragging" : ""}
```

Comment:
The `dragging` prop on `SVGLine` was only used in this one place, so I opted to remove it and set the className directly. It seems like `SVGLine` shouldn't know about dragging states anyway, just based on its name. Dragging, and the CSS classes related to it, are application-level concepts, not SVG concepts.